electronics/details/details_flashing.md

```diff
@@ -0,0 +1,23 @@
+# Details Flashing
+
+This document describes how to flash the firmware on the TI LaunchPad Development Kit.
+
+## Step 1 - Remove jumpers
+
+After unpacking the LaunchPad, 
+* Remove the jumpers JP1, JP2, JP4, JP5
+* Enable the jumper JP6
+
+## Step 2 - Install software to flash the firmware
+
+To be written.
+* How to install Code Composer
+* How to clone the required code
+
+## Step 3 - Flash the firmware
+
+Make sure the LaunchPad is powerd through motor boaster packs.
+
+* Toggle the switches to 1-2-3 pointing all towards the USB connector
+* Flash the firmware through Code Composer
+* Toggle the switch 3 back to the original configuration
```

Comment:
We already have a more detailed description for that in the internal wiki ("How to run a CCS Project on the LaunchPad"), I think that could simply be moved here (including the images of the switches).